todo.sh

```diff
@@ -1211,9 +1212,11 @@ case $action in
 		echo "TODO: $item deprioritized."
 	    fi
 	else
-	    echo "TODO: $item is not prioritized."
+	    echo >&2 "TODO: $item is not prioritized."
+	    status=1
```

Comment:
Is it necessary here to leave the whole process with failure exit status? Maybe you could test if other `$item`s succeed?

Reply:
For me, failure is the inability to do what I was told, even if I partially succeeded. The user still has the action's output to see which items succeeded and which didn't; the exit status is more of interest from a scripting perspective; the shell doesn't visualize the exit status by default (but I color the following prompt red if there was an error). And from that scripting perspective, it's more interesting whether the whole command succeeded, not whether it got partially done. (And you can always use a loop and invoke the command with each item separately to obtain that information.)